macros/src/main/scala/packaged/macros/InnerObjectMacro.scala

```diff
@@ -0,0 +1,20 @@
+package packaged.macros
+import scala.gestalt._
+
+/**
+  * Created by valdis on 17.28.2.
+  */
```

Comment:
Sorry about that, I should really find the setting to disable that. Removed in next commit.